conf/establishersCompanyDirector.routes

```diff
@@ -0,0 +1,164 @@
+POST       /:establisherIndex/company/director/:directorIndex/national-insurance-number                             controllers.register.establishers.company.director.DirectorHasNINOController.onSubmit(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+GET        /:establisherIndex/company/director/:directorIndex/change/national-insurance-number                      controllers.register.establishers.company.director.DirectorHasNINOController.onPageLoad(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/change/national-insurance-number                      controllers.register.establishers.company.director.DirectorHasNINOController.onSubmit(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+GET        /:establisherIndex/company/director/:directorIndex/:mode/national-insurance-number/:srn                  controllers.register.establishers.company.director.DirectorHasNINOController.onPageLoad(mode: Mode, establisherIndex: Index, directorIndex: Index, srn: Option[String])
+POST       /:establisherIndex/company/director/:directorIndex/:mode/national-insurance-number/:srn                  controllers.register.establishers.company.director.DirectorHasNINOController.onSubmit(mode: Mode, establisherIndex: Index, directorIndex: Index, srn: Option[String])
+
+GET        /:establisherIndex/company/director/:directorIndex/reason-for-no-national-insurance-number               controllers.register.establishers.company.director.DirectorNoNINOReasonController.onPageLoad(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/reason-for-no-national-insurance-number               controllers.register.establishers.company.director.DirectorNoNINOReasonController.onSubmit(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+GET        /:establisherIndex/company/director/:directorIndex/change/reason-for-no-national-insurance-number        controllers.register.establishers.company.director.DirectorNoNINOReasonController.onPageLoad(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/change/reason-for-no-national-insurance-number        controllers.register.establishers.company.director.DirectorNoNINOReasonController.onSubmit(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
```

Comment:
DirectorNoNinReason routes are needed in UpdateMode

Reply:
done